### ISSUE TITLE
Something (WordFence?) Prevents Entry Editing

### ISSUE THREAD

**Issue by ChrisHinde** [CalderaWP/Caldera-Forms#1528]
## Do You Need Immediate Help?
As soon as possible (using the free plugin/can't buy support [non-commercial use]).

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 4.7.4PHP Version: 7.0.17MySQL Version: 5.7.17Caldera Forms Version: 1.5.0.8WP_DEBUG:

## Does Your Issue Presist When You Disable All Other Plugins and Switch To The Default Theme?
Yes!

## What Is The Unexpected Beahviour?
The edit entry form isn't shown!
![cf_editentry](https://cloud.githubusercontent.com/assets/7074726/25298661/3d3f7318-26f7-11e7-8be7-acfaa74daa98.png)

## What PHP Errors Have You Logged While Reproducing This Bug?
403 Forbidden

## What JavaScript Errors Have You Seen While Reproducing This Bug?
Request URL:http://segolstorp.se/cf-api/CF58f8d8e41cf2d/169/?cf-api=CF58f8d8e41cf2d&modal=view_entry&modalElement=div&modalTitle=Editing+Entry++%23169&modalButtons=Save+Changes%7C%7B%22data-for%22%3A%22%23view_entry_baldrickModalBody+.caldera_forms_form%22%7D%7Cright+button-primary%3BView+Entry%7C%7B%22data-for%22%3A%22.view-entry-btn.current-view%22%7D%7C&method=get&loadClass=spinner&activeClass=current-edit&request=http%3A%2F%2Fsegolstorp.se%2Fcf-api%2FCF58f8d8e41cf2d%2F169%2F%3Fcf-api%3DCF58f8d8e41cf2d&modalHeight=auto&modalWidth=700&group=editentry&form=CF58f8d8e41cf2d&entry=169&static=true&_=1492815400599
Request Method:GET
Status Code:403 Forbidden
Remote Address:194.9.94.100:80
Referrer Policy:no-referrer-when-downgrade
Response Headers
view source
Connection:keep-alive
Content-Length:229
Content-Type:text/html; charset=iso-8859-1
Date:Fri, 21 Apr 2017 22:59:05 GMT
Server:nginx/1.10.2
Request Headers
view source
Accept:*/*
Accept-Encoding:gzip, deflate, sdch
Accept-Language:sv-SE,sv;q=0.8,en-US;q=0.6,en;q=0.4
Connection:keep-alive
Cookie:wordpress_test_cookie=
DNT:1
Host:segolstorp.se
Referer:http://segolstorp.se/wp-admin/admin.php?page=caldera-forms-pin-CF58f8d8e41cf2d
User-Agent:Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36
X-Requested-With:XMLHttpRequest
Query String Parameters
view source
view URL encoded
cf-api:CF58f8d8e41cf2d
modal:view_entry
modalElement:div
modalTitle:Editing Entry  #169
modalButtons:Save Changes|{"data-for":"#view_entry_baldrickModalBody .caldera_forms_form"}|right button-primary;View Entry|{"data-for":".view-entry-btn.current-view"}|
method:get
loadClass:spinner
activeClass:current-edit
request:http://segolstorp.se/cf-api/CF58f8d8e41cf2d/169/?cf-api=CF58f8d8e41cf2d
modalHeight:auto
modalWidth:700
group:editentry
form:CF58f8d8e41cf2d
entry:169
static:true
_:1492815400599

**Comment by Shelob9** [comment]
@ChrisHinde Thanks for using Caldera Forms.

This 403 error is caused by something on your server. It is most likely a security plugin, web application firewall like SiteLock, or proxy like Cloudflare treating this valid request as something malicious, which it is not.

Try temporarily disabling your security plugins, WAF or proxy or contacting your host. If you need help on our end, please open a support ticket at https://calderaforms.com/support

**Comment by Shelob9** [comment]
Looks like another use has this problem and it was related to WordFence: https://wordpress.org/support/topic/entries-not-showing-in-dashboard/#post-9043252

**Comment by amsgator** [comment]
For me, I was getting 403 for admin-ajax.php. I was able to determine it was caused by the WOrdfence WAF. The user.ini file created by the Wordfence WAF contains the following:

```
; Wordfence WAF
auto_prepend_file = '/home/username_here/public_html/wordfence-waf.php'
; END Wordfence WAF
```

At the time I had an issue, I had a WordPress installation in a subdirector, say /home/username_here/public_html/sub_directory

Apparently the WAF was returning 403 for certain parts of the subdirectory installation. Once I setup the WAF on the subdirectory installation everything was fixed. I'm not sure if your situation is similar, but that was my issue and how I solved it.
